src/Engine/GUI/Workspaces/OrbitalWorkspace.cpp

```
@@ -163,20 +166,32 @@ void OrbitalWorkspace::loadSimulationConfig(const std::string &configPath) {
+	if (FilePathUtils::PathExists(configPath)) {
+		m_simulationConfigPath = configPath;
+		m_simulationScriptData = FilePathUtils::ReadFile(configPath);
 
-	m_eventDispatcher->dispatch(RequestEvent::InitSession{
-		.simulationFilePath = configPath
-	});
+		m_eventDispatcher->dispatch(RequestEvent::InitSession{
+			.simulationFilePath = configPath
+		});
+	}
+	else {
```

Clang (14.0.6)

[note] else branch starts here

(clang)